pkg/debian/salt-master.preinst

```diff
@@ -1,15 +1,25 @@
 case "$1" in
   install|upgrade)
-    [ -z "$SALT_HOME" ] && SALT_HOME=/opt/saltstack/salt
-    [ -z "$SALT_USER" ] && SALT_USER=salt
-    [ -z "$SALT_NAME" ] && SALT_NAME="Salt"
-    [ -z "$SALT_GROUP" ] && SALT_GROUP=salt
-    PY_VER=$(/opt/saltstack/salt/bin/python3 -c "import sys; sys.stdout.write('{}.{}'.format(*sys.version_info)); sys.stdout.flush();")
 
-    # Reset permissions to fix previous installs
-    find ${SALT_HOME} /etc/salt /var/log/salt /var/cache/salt /var/run/salt \
-        \! \( -path /etc/salt/cloud.deploy.d\* -o -path /var/log/salt/cloud -o -path /opt/saltstack/salt/lib/python${PY_VER}/site-packages/salt/cloud/deploy\* \) -a \
-        \( -user ${SALT_USER} -o -group ${SALT_GROUP} \) -exec chown root:root \{\} \;
+    case "$2" in
+      # if upgrading from 3006.0/3006.1/3006.2 reset permissions
+      # for salt user - issue 64193 and 65264
+
+      3006.0|3006.1|3006.2)
```

Comment:
these are pretty old versions of Salt , what about later versions eg 3006.7, 3007.0 etc.
Can this be done a different way ? that is not so version specific ?